web/src/lib/utils/duplicate-utils.spec.ts

```diff
@@ -1,5 +1,11 @@
-import { suggestDuplicate } from '$lib/utils/duplicate-utils';
+import { DuplicateSelection } from '$lib/utils/duplicate-utils';
 import type { AssetResponseDto } from '@immich/sdk';
+import type { SourcePreference } from '$lib/stores/duplicate-tie-preferences';
+
+const duplicateSelector = new DuplicateSelection();
+
+const suggestDuplicate = (assets: AssetResponseDto[], pref = undefined) =>
```

Comment:
What's the type of `pref`? Just undefined? Then just omit it? (also, we try to avoid abbreviations => `preference`)